src/main/java/com/epam/wl/entities/UserOrder.java

```diff
@@ -7,7 +7,12 @@
 @Data @AllArgsConstructor
```

Comment:
It would be better to start each annotation from new line

Reply:
Fixed